tests/foreman/installer/test_installer.py

```diff
@@ -1103,8 +1307,9 @@
     u'--reset-puppet-version',
     u'--scenario',
     u'--skip-checks-i-know-better',
+    u'--skip-puppet-version-check',
     u'--upgrade',
-    u'--upgrade-puppet',
+    u'--upgrade-mongo-storage-engine',
```

Comment:
@alda519 Is this option is regarding new mongo upgrade feature?

Reply:
yes, it is
see https://bugzilla.redhat.com/show_bug.cgi?id=1593253